tests/pytorch_tests/model_tests/feature_models/activation_bias_correction_test.py

```diff
@@ -86,74 +87,39 @@ def forward(self, x):
-        super().__init__(unit_test)
-
-    def get_quantization_config(self):
-        # A large value is assigned to the activation bias correction threshold to enable threshold filtering,
-        # which adjusts the bias correction values to zero.
-        return mct.core.QuantizationConfig(weights_bias_correction=False,
-                                           weights_second_moment_correction=False,
+                                           shift_negative_activation_correction=False,
                                            activation_bias_correction=True,
-                                           activation_bias_correction_threshold=1e9)
+                                           activation_bias_correction_threshold=
+                                           self.activation_bias_correction_threshold)
 
     def create_networks(self):
-        return ActivationBiasCorrectionPadNet()
+        return self.model
 
     def compare(self, quantized_model, float_model, input_x=None, quantization_info=None):
         bias = float_model.linear_layer.bias.cpu().detach().numpy()
         bias_after_activation_bias_correction = quantized_model.linear_layer.layer.bias.cpu().detach().numpy()
-        self.unit_test.assertTrue(np.array_equal(bias, bias_after_activation_bias_correction),
-                                  msg=f"Error in activation bias correction: expected no change in the bias value in "
-                                      f"case of activation_bias_correction_threshold 1e9.")
 
-
-class BaseActivationBiasCorrectionReshapeNetTest(BaseActivationBiasCorrectionTest):
-    def __init__(self, unit_test):
-        super().__init__(unit_test)
-
-    def create_networks(self):
-        return ActivationBiasCorrectionReshapeNet()
+        if getattr(float_model.linear_layer, KERNEL_SIZE, None) in [None, 1, (1, 1)]:
+            if self.activation_bias_correction_threshold > 1e8:
+                self.unit_test.assertTrue(np.array_equal(bias, bias_after_activation_bias_correction),
+                                          msg=f"Error in activation bias correction: expected no change in the bias "
+                                              f"value in case of activation_bias_correction_threshold "
+                                              f"{self.activation_bias_correction_threshold}.")
+            else:
+                self.unit_test.assertFalse(np.array_equal(bias, bias_after_activation_bias_correction),
+                                           msg=f"Error in activation bias correction: expected a change in the bias "
+                                               f"value.")
+        else:
+            self.unit_test.assertTrue(np.array_equal(bias, bias_after_activation_bias_correction),
+                                      msg=f"Error in activation bias correction: expected no change in the bias value "
+                                          f"in case of conv with kernel different than 1 or (1, 1).")
```

Comment:
I think you should add an output compare, so that we make sure that the quantized model is built correctly.